### ISSUE TITLE
Enabling 'Realtime Audio' option makes Games no longer work

### ISSUE THREAD

**Issue by ghost** [Lime3DS/Lime-3DS-Emulator#362]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Lime3DS 2116, Lime3DS 2117

### Description of Issue

After Enabling the 'Realtime Audio' option games are either get stuck on Black screen or not boot at all and stays on infinite Loading. Games no longer can be played as usual after Enabling that option.This issue is first seen on Lime3DS version 2116, and unfortunately it also affected on Lime3DS version 2117

### Expected Behavior

Games should work fine after Enabling Realtime Audio option

### Reproduction Steps

Enable 'Realtime Audio' from the Audio setting, Open a Game.

### Log File

[lime3ds_log.txt](https://github.com/user-attachments/files/16626965/lime3ds_log.txt)

### System Configuration

CPU: Octa-core (4x2.35 GHz Cortex-A53 & 4x1.8 GHz Cortex-A53)
GPU/Driver: PowerVR GE8320
RAM: 4GB
OS: Android

**Comment by OpenSauce04** [comment]
Have you tried disabling cheats?
Have you tried multiple games?

**Comment by ChldGamer** [comment]
Yes, I tried to open multiple games and disable all cheats, but still no games worked.

**Comment by ghost** [comment]
> Have you tried disabling cheats?
> Have you tried multiple games?

Yes, I tried to open multiple games and disable all cheats, but still no games worked.

**Comment by ghost** [comment]
@OpenSauce04 Dev, could you please address this issue?

**Comment by OpenSauce04** [comment]
I only have so much time in the day, other priorities are taking precedent. Please be patient.

**Comment by OpenSauce04** [comment]
Looking at the log, it appears that this issue was caused due to a broken pirated ROM.
